secator/tasks/prompt.py

```
@@ -0,0 +1,45 @@
+import click
+
+from secator.decorators import task
+from secator.output_types import Tag
+from secator.runners import PythonRunner
+from secator.rich import console
+
+
+@task()
+class prompt(PythonRunner):
+    """Prompt the user for a CIDR range."""
+    output_types = [Tag]
```

Ruff (0.14.8)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)